colour/models/din99.py

```diff
@@ -85,37 +100,41 @@ def Lab_to_DIN99(Lab, k_E=1, k_CH=1):
     >>> Lab_to_DIN99(Lab)  # doctest: +ELLIPSIS
     array([ 53.2282198...,  28.4163465...,   3.8983955...])
     """
+    p = DIN99_METHODS[str(method)]
 
     L, a, b = tsplit(to_domain_100(Lab))
 
-    cos_16 = np.cos(np.radians(16))
-    sin_16 = np.sin(np.radians(16))
+    cos = np.cos(np.radians(p[2]))
+    sin = np.sin(np.radians(p[2]))
```

Comment:
`cos` and `sin` feel dangerous here, should we add a `_c` for coefficient suffix? e.g. `cos_c`